src/components/pages/UserPage.js

```diff
@@ -3,11 +3,18 @@ import { Link, Redirect } from 'react-router-dom';
 import D3Chart from '../chart/D3Chart';
 import API from '../../utils/api';
 import PropTypes from 'prop-types';
+import { atom, useRecoilState } from 'recoil';
+
+const userState = atom({
```

Comment:
nit - could move this to a new folder `recoil/atoms` and import userState

Also would `userRepoState` or something like that be more specific?

Reply:
I do like userRepoState